File: assets/samples/domain.availability/password/net/DomainAvailabilityApi.cs
using System;
using System.Net;
using System.IO;

class Program {
    public const string DOMAIN = "example.com";
    public const string PASSWORD = "your domain availability api password";
    public const string USERNAME = "your domain availability api username";

    static void Main() {
        string url = "http://www.whoisxmlapi.com/whoisserver/WhoisService?"
                   + $"domainName={DOMAIN}&username={USERNAME}"
                   + $"&password={PASSWORD}&cmd=GET_DN_AVAILABILITY"
                   + $"&getMode=DNS_AND_WHOIS";
        string whoisData = string.Empty;

        HttpWebRequest req = (HttpWebRequest)WebRequest.Create(url);

        using (HttpWebResponse response = (HttpWebResponse)req.GetResponse())
        using (Stream stream = response.GetResponseStream())
        using (StreamReader reader = new StreamReader(stream)) {
            whoisData = reader.ReadToEnd();
        }
        Console.WriteLine(whoisData);
    }
}

File: assets/samples/domain.availability/password/java/DomainAvailabilityApi.java
public class DomainAvailabilityApi {
    public static final String DOMAIN = "example.com";
    public static final String PASSWORD =
        "Your domain availability api password";
    public static final String USERNAME =
        "Your domain availability api username";

    public static void main(String[] args) throws Exception {
        String url = "http://www.whoisxmlapi.com/whoisserver/WhoisService"
                   + "?domainName=" + DOMAIN + "&userName=" + USERNAME
                   + "&password=" + PASSWORD + "&cmd=GET_DN_AVAILABILITY"
                   + "&getMode=DNS_AND_WHOIS";
        try (java.util.Scanner s =
                new java.util.Scanner(new java.net.URL(url).openStream())) {
            System.out.println(s.useDelimiter("\\A").next());
        }
    }
}